### PR TITLE
Milan connectivity

### PR DESCRIPTION
**Pull request by Garlyle**

### CODE REVIEW

File: app/src/main/java/com/greenfox/opal/gitinder/MainActivity.java [hunk 1]
```diff
@@ -45,9 +51,29 @@ protected void onCreate(Bundle savedInstanceState) {
...
     checkLogin();
   }
 
+  private void checkConnection() {
+    ConnectivityManager cm = (ConnectivityManager)getApplicationContext().getSystemService(Context.CONNECTIVITY_SERVICE);
+    NetworkInfo activeNetwork = cm.getActiveNetworkInfo();
+    boolean isConnected = activeNetwork != null && activeNetwork.isConnected();
+    if (!isConnected) {
+      AlertDialog alertDialog = new AlertDialog.Builder(this).create();
+      alertDialog.setTitle("Info");
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/opal/gitinder/MainActivity.java]: @Garlyle use string resource

File: app/src/main/java/com/greenfox/opal/gitinder/MainActivity.java [hunk 2]
```diff
@@ -45,9 +51,29 @@ protected void onCreate(Bundle savedInstanceState) {
...
+      alertDialog.setMessage("Internet not available, check your internet connectivity and try again");
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/opal/gitinder/MainActivity.java]: @Garlyle use string resource

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/greenfox/opal/gitinder/MainActivity.java (hunk 2 above)
Comment (Garlyle) [reply app/src/main/java/com/greenfox/opal/gitinder/MainActivity.java]: @adamgyulavari done

**Review by adamgyulavari — APPROVED** [review]

**Comment by adamgyulavari** [comment]
@Garlyle resolve conflicts

**Comment by adamgyulavari** [comment]
@Garlyle resolve conflicts

**Pull request merged**